tools/clang_tools/support/BUILD.prebuilt

```diff
@@ -1,7 +1,7 @@
-# Clang 10.0 library pre-built Bazel.
+# Clang 11.0 library pre-built Bazel.
 #
 # This file was mostly manually assembled (with some hacky Python scripts) from
-# clang+llvm-10.0.0-x86_64-linux-gnu-ubuntu-18.04.tar.xz and corresponding
+# clang+llvm-11.0.0-x86_64-linux-gnu-ubuntu-16.04.tar.xz and corresponding
```

Comment:
Are we going backward from 18.04 to 16.04 here?

Reply:
Either ubuntu 16.04 and 20.04 has prebuild 11.0.0 https://releases.llvm.org/download.html
Prebuild 11.0.0-ubuntu16.04 is running fine on my ubuntu 20.04 except lldb-11.
I guess we don't need lldb-11 during the build.

Reply:
Yeah that's because they don't have build for 18.04 and 16.04 would just work fine in 18.04.

Reply:
Fair enough; prefer going forward than back all things equal but not if there are complications.